css-ui-4/Overview.bs

```diff
@@ -2199,6 +2203,42 @@ that are typical of the element whose appearance it acquires.
 	is also not changed by the 'appearance' property.
 </div>
 
+<h3 id="input-security">Obscuring sensitive input: the 'input-security' property</h3>
```

Comment:
Use the same linebreaking as the rest of the spec, putting the text content on a newline.

Reply:
Fixed in 1dd4278.